pkg/controllers/disruption/eventual.go

```diff
@@ -70,9 +72,9 @@ func (d *Drift) ComputeCommand(ctx context.Context, disruptionBudgetMapping map[
 		}
 		// If there's disruptions allowed for the candidate's nodepool,
 		// add it to the list of candidates, and decrement the budget.
-		if disruptionBudgetMapping[candidate.nodePool.Name][d.Reason()] > 0 {
+		if disruptionBudgetMapping[candidate.nodePool.Name] > 0 {
 			empty = append(empty, candidate)
```

Comment:
Calling this "empty" I don't think is correct anymore

Reply:
Why is that? we are checking if the candidate is empty, no?

Reply:
Yep, just misread this section.